.github/workflows/issues_metrics.yml

```diff
@@ -0,0 +1,50 @@
+name: Monthly issue metrics
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '3 2 1 * *'
+
+permissions:
+  issues: write
+
+jobs:
+  build:
+    name: issue metrics
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Get dates for last month
+      shell: bash
+      run: |
+        # Get the current date
+        current_date=$(date +'%Y-%m-%d')
+
+        # Calculate the previous month
+        previous_date=$(date -d "$current_date -1 month" +'%Y-%m-%d')
+
+        # Extract the year and month from the previous date
+        previous_year=$(date -d "$previous_date" +'%Y')
+        previous_month=$(date -d "$previous_date" +'%m')
+
+        # Calculate the first day of the previous month
+        first_day=$(date -d "$previous_year-$previous_month-01" +'%Y-%m-%d')
+
+        # Calculate the last day of the previous month
+        last_day=$(date -d "$first_day +1 month -1 day" +'%Y-%m-%d')
+
+        echo "$first_day..$last_day"
+        echo "last_month=$first_day..$last_day" >> "$GITHUB_ENV"
+
+    - name: Run issue-metrics tool
+      uses: github/issue-metrics@v2
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SEARCH_QUERY: 'repo:kedro-org/kedro is:issue created:${{ env.last_month }} -reason:"not planned"'
+
+    - name: Create issue
+      uses: peter-evans/create-issue-from-file@v4
+      with:
+        title: Monthly issue metrics report
+        token: ${{ secrets.GITHUB_TOKEN }}
+        content-filepath: ./issue_metrics.md
```

Comment:
Where does this file come from?

Reply:
https://github.com/github/issue-metrics#multiple-repositories-example

If you look at the example, it generated by github/issue-metrics@v2

Reply:
I don't know where can we put the output CSV to. It's possible but not trivial. If you try to copy it manually from an issue and do `pandas.read_clipboard` it would parse them correctly.

If we really want to automate it we can have pandas parsing the markdown, but I don't know if it's worth to do it.

Reply:
Fair enough 👍🏽